tests/python/test_comm_hooks_fsdp.py

```diff
@@ -541,10 +556,11 @@ def test_gossip_grad_communication_cube(self, sharding_strategy):
         # topology for ease of computation, thus mahually hardcode 1 topology
         state.topologies = itertools.cycle([master_ranks])
         state.cur_topology = next(state.topologies)
+
         # There will be only `state.gossip_period` different communication peer changes,
         # new iteration -> new peer.
         # Thus, only checking `state.gossip_period` possible steps.
-        for _ in range(state.gossip_period):
+        for _ in range(5):  # state.gossip_period):
```

Comment:
why did we change this?

Reply:
forgot to change back, in a new version of this pr will bring it back